examples/fizz-buzz.ktn

```diff
@@ -0,0 +1,26 @@
+define buzzes(Int32 -> +IO +Fail):
+    -> n;
+    n 5 (%) 0 (=)
```

Comment:
You can wrap these in `{ … } with (+Fail)` (or `do (with (+Fail)) { … }` if you prefer) to unsafely assert that the divisions won’t fail, and thereby remove the `+Fail` constraint on `buzzes` and `loop`.

There’s no real need for these operator calls to be postfix unless you’re taking advantage of composition—but by writing them in postfix you’ve revealed a good opportunity to factor out `(%) 0 (=)` into a definition called e.g. `divisible`.

It would be straightforward to remove the `n` variable as well (with `dup dup` to get a couple copies for the tests, and `drop` in the conditional branches where you ignore the value, e.g. `drop "Fizz"`) but that’s a matter of style and not terribly important.